Core/GUI/GameTooltipHooks.lua

```diff
@@ -510,6 +510,23 @@ local function processItem(id)
 	end
 end
 
+local function onTooltipSetItem(tooltip, tooltipData)
+	if tooltip ~= _G.GameTooltip and tooltip ~= _G.ItemRefTooltip then
+		return
+	end
+
+	local _, itemLink = tooltip:GetItem()
+	if type(itemLink) ~= "string" then
+		return
+	end
+
+	local id = itemLink:match("item:(%d+):")
+	processItem(tonumber(id), tooltip)
+end
+
+_G.TooltipDataProcessor.AddTooltipPostCall(_G.Enum.TooltipDataType.Item, onTooltipSetItem)
+
+--[[
```

Comment:
I just noticed this part - is it safe to remove them? If yes, why comment it out? If no, breaking change?

In other words: Does the `TooltipDataProcessor` still work for all those other scenarios? (I'd have to research it)

Reply:
I left it commented out so it would be easier to re-add if it turned out to not work for _all_ areas.

I personally have not experienced any ill effects 🤔

Reply:
That's what Git is used for :P As far as I understand, this `TooltipDataType` should work for all item tooltips. I suppose we can just remove the comments and see if anyone complains about missing a tooltip somewhere (seems unlikely).

Reply:
Cheeky 😂 it's still slightly harder to revert part of a commit! 

But point taken, I can adjust it as soon as I get home 👍